frontend/app/src/composables/balances.ts

```diff
@@ -236,3 +238,32 @@ export const useExchanges = () => {
     connectedExchanges
   };
 };
+
+export const setupEnsNames = () => {
+  const store = useStore();
+
+  const isEnsEnabled = computed<boolean>(() => {
```

Comment:
Do we use the ens part in the Vuex store? or would it make sense to make a new pinia store?

Reply:
Yeah you're right, I'm moving it!

Reply:
Updated, can you check again? @kelsos